requirements.txt

```diff
@@ -4,3 +4,4 @@ py3k-bcrypt==0.3
 stripe==1.11.0
 pytz==2014.4
 pep8==1.6.2
+pytest==2.7.2
```

Comment:
Sideboard ships with an older version of py.test and I wanted the later one.  I should probably update Sideboard to require this version as well.